### DISCUSSION TITLE
Proposal for Summer program

### DISCUSSION THREAD

**Discussion by Inocent-web** [MDAnalysis/mdanalysis#4526] [GSoC Discussions]
Hello 
I am a student from the College of St Scholastica majoring in chemistry and Computer Science. I am currently doing research in computational chemistry, and I have also great python programming experience. Some of my skills include using Linux Terminal Command, Visual Molecular Dynamics program, Proficient with VI, Molecular
Dynamics Simulations, Monté Carlo simulations and Alphafold2.May you help me with details on writing the proposal for the application.

**Comment by RMeli** [comment]
Thank you for participating in GSOC 2024 with us. Whether you were selected as a GSoC contributor or not, you are a valued member of the MDAnalysis community and we are excited to work with you outside of the GSoC program as well!
